XML/XML.sublime-syntax

```diff
@@ -20,89 +21,62 @@ scope: text.xml
 variables:
   # This is the full XML Name production, but should not be used where namespaces
   # are possible. Those locations should use a qualified_name.
-  name: '[[:alpha:]:_][[:alnum:]:_.-]*'
+  name: '(?i)[[:alpha:]:_][[:alnum:]:_.-]*'
```

Comment:
`:alnum:` and `:alpha:` already includes upper and lower case, so the `(?i)` is redundant, plus will affect the rest of any regex patterns that use these variables. http://www.regular-expressions.info/posixbrackets.html